stacks/op-scim-bridge/yaml/op-scim-bridge.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     spec:
       containers:
       - name: op-scim-bridge
-        image: 1password/scim:v1.2.1
+        image: 1password/scim:v1.3.0
```

Comment:
I'm seeing image pull issues for this image; does it exist?

Reply:
Thanks Paul. You are correct. This version is oddly named. It should be `1password/scim:v1.3.0-2`. I think I was accidentally testing with a cached older version actually titled `1password/scim:v1.3.0` which never saw the light of day.

Reply:
I believe I have fixed that correctly. After pruning my old images and re-rendering the DO marketplace stack, it deploys correctly to my DO cluster. 

Let me know how that looks.